examples/simple-game-server/Makefile

```
@@ -0,0 +1,46 @@
+# Copyright 2017 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Makefile for building a simple game-server
+#
+
+#  __     __         _       _     _
+#  \ \   / /_ _ _ __(_) __ _| |__ | | ___ ___
+#   \ \ / / _` | '__| |/ _` | '_ \| |/ _ \ __|
+#    \ V / (_| | |  | | (_| | |_) | |  __\__ \
+#     \_/ \__,_|_|  |_|\__,_|_.__/|_|\___|___/
+#
+
+REPOSITORY = gcr.io/agones-images
+
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+project_path := $(dir $(mkfile_path))
+server_tag = $(REPOSITORY)/game-server:0.22
```

I assume we could start counting from the start:
```suggestion
server_tag = $(REPOSITORY)/game-server:0.1
```